src/main/resources/lang/quilt_loader_es_AR.properties

```diff
@@ -0,0 +1,132 @@
+# It is Argentinian Spanish (es_AR).
+
+# Node names
+gui.text.classpath=Trayectoria de clases
+gui.text.floating_mods_from_plugins=Mods flotantes desde plugins
+gui.text.arg_mods=Argumentos del cargador
+# General text
+gui.text.plugin=plugin: '%s'
+gui.text.loaded_by_plugin=cargado por plugin: %s
+gui.text.id=id: '%s'
+gui.text.version=versión: '%s'
+gui.text.providing=proveyendo: '%s'
+gui.text.game_version_match=Iniciando, ya que coincide con la %s
+gui.text.game_version_mismatch=No se va a iniciar, ya que no coincide con la %s
+gui.prefix.no_parent_dir=*No hay directoro pariente* %s
+gui.prefix.no_parent_file=*No hay archvo pariencte* %s
+info.root_mod_loaded_from=%esta cargado desde %s
+tab.messages=Errores
+tab.file_list=Archivos
+tab.mod_list=Mods
+msg.load_state=Estado de carga del mod (para depuración - ¡Esto significa que ha funcionado!)
+msg.load_state.warns=Estado de carga del mod (para depuración - ¡Esto significa que ha funcionado!)\n% advertencias.
+# Buttons
+button.view_file=Mirar %s
+button.edit_file=Editar %s
+button.open_crash_report=Abrir reporte de choque
+button.copy_crash_report=Copiar reporte de choque
+button.copy_section=Abrir reporte de choque
+button.mod_issue_tracker=Abrir el Rastreador de Problemas de %s
+button.quilt_loader_report=Abrir la página de informe de Quilt Loader
```

Comment:
What does this look like in the actual GUI?

Reply:
So that particular string isn't referenced any more :/

I guess I should have removed it from the lang files when we removed direct links to the github issues page?